README.md

```diff
@@ -1,2 +1,15 @@
 # nva-search-api
 A common search API for NVA across resources
+
+
+Structure wanted in response from the search API
+```JSON
+{
+"createdDate": "2020-08-10T07:56:27.232208Z",
```

Comment:
better to create a resource file that is updated by a test and refer to it here.

Reply:
Seems to be no usable way to include files in a markdow, but has added a sample file to be used in tests later on...